docs: update pull requests docs to include a how to ask section (#6872)
typescript-eslint/typescript-eslint: docs/contributing/Pull_Requests.mdx

Once you've addressed all our feedback by making code changes and/or started a followup discussion, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews#re-requesting-a-review) from each maintainer whose feedback you addressed.

Once the feedback is addressed, and the PR is approved, we'll ensure the branch is up to date with `main`, and merge it for you.

> ### How to Ask

Reviewer: Suggestion to make it a bit more clear. What do you think about going with one of these?

```suggestion
### Asking for Help
```

```suggestion
### Asking Questions
```

Author: That's true. I agree. Asking Questions is a much more clearer title.

> If you need help and/or have a question, posting a comment in the PR is a great way to do so. Once you do, one of us will drop by and help when we can.

Reviewer: A couple of proofreading thoughts:

* Folks have a habit of explicitly tagging us - part of #6584's intent was to get them to not do that
* The rest of this document generally puts sentences on their own lines (see reasoning in https://github.com/JoshuaKGoldberg/sentences-per-line)

```suggestion
If you need help and/or have a question, posting a comment in the PR is a great way to do so.
There's no need to tag anybody individually.
One of us will drop by and help when we can.
```

Author: Thank you for providing context! That 100% makes sense, I agree adding the sentence in not needing to tag anyone makes it more clearer.

Once you've addressed all our feedback by making code changes and/or started a followup discussion, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews#re-requesting-a-review) from each maintainer whose feedback you addressed.

Once the feedback is addressed, and the PR is approved, we'll ensure the branch is up to date with `main`, and merge it for you.

### Asking Questions

> If you need help and/or have a question, posting a comment in the PR is a great way to do so.

Reviewer: (lil grammar nit since the heading is updated)

```suggestion
If you need help and/or have a question, posting a comment in the PR is a great way to ask.
```
